app/views/users/_form.html.haml

```diff
@@ -0,0 +1,43 @@
+= form_for @user, url: path do |f|
+  .container-fluid
+    .row
+      .col-lg-6
+        .form-group
+          = f.label :first_name
+          = f.text_field :first_name, class: 'form-control'
+        .form-group
+          = f.label :last_name
+          = f.text_field :last_name, class: 'form-control'
+        .form-group
+          = f.label :spire
+          = f.email_field :spire, size: 25,
+           placeholder: '12345678@umass.edu', class: 'form-control'
+        .form-group
+          = f.label :email
+          = f.email_field :email, size: 30, class: 'form-control'
+        .form-group
+          = f.label :phone
+          = f.phone_field :phone,
+           placeholder: '+14135450056', class: 'form-control'
+      .col-lg-6
+        .form-group
+          = f.label :rosters
+          = f.select :rosters,
+           options_from_collection_for_select(Roster.all, :id, :name,
+            selected: selected_rosters(@user, @roster)), {}, multiple: true, class: 'form-control'
+        %div
+          = f.check_box :reminders_enabled
+          = f.label :reminders_enabled,
+           'Receive email reminders of upcoming assignments?'
+        %div
+          = f.check_box :change_notifications_enabled
+          = f.label :change_notifications_enabled,
+           'Receive notifications when your assignments are changed?'
+        - if @current_user.admin_in? @roster
+          %div
+            = f.fields_for @user.membership_in(@roster) || :membership do |g|
+              %div
+                = g.check_box :admin
+                = g.label :admin, "Admin in #{@roster.name}?"
```

Comment:
I know you didn't add this functionality, but we probably shouldn't allow the only admin of a roster to make themselves not the admin of said roster.

Reply:
#170